### DISCUSSION TITLE
Cannot read property 'oidc' of undefined

### DISCUSSION THREAD

**Discussion by jcharnet** [ory/kratos#1301] [Q&A] [answered]
Just started playing around with Kratos, and following the Quickstart guide, after I run the docker-compose command and navigate to the url 127.0.0.1:4455/dashboard I'm getting this error in the logs:

> kratos-selfservice-ui-node_1  | TypeError: Cannot read property 'oidc' of undefined
> kratos-selfservice-ui-node_1  |     at Object.exports.methodConfig (/usr/src/app/lib/helpers.js:36:22)
> kratos-selfservice-ui-node_1  |     at /usr/src/app/lib/routes/login.js:68:76
> kratos-selfservice-ui-node_1  |     at processTicksAndRejections (internal/process/task_queues.js:97:5)
> kratos-selfservice-ui-node_1  | GET /auth/login?flow=231324c5-5ee4-4840-9cb7-5b325561d8ad 500 11364 - 24.895 ms
> kratos-selfservice-ui-node_1  | GET /index.css 304 - - 0.912 ms
> kratos-selfservice-ui-node_1  | GET /typography.css 304 - - 1.979 ms
> kratos-selfservice-ui-node_1  | GET /form.css 304 - - 1.602 ms
> kratos-selfservice-ui-node_1  | GET /auth.css 304 - - 1.892 ms

I have tried this using the master branch and the v0.6.0-alpha.1.pre.2 tag.
I'm running the following command to start kratos:
`docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate`

Any ideas. what's happening?

Thank you,

Joao

**Comment by vinckr** [comment]
The `v0.6.0-alpha.1.pre.2` release is very fresh, so something is not working as intended; 
but I cant say what exactly at the moment. 
It will most likely be resolved in the coming week.

In the meantime: 
Try it out with
`git checkout v0.5.5-alpha.1`
Quickstart is working in that version.

**Reply by aeneasr** [reply]
A non-pre release is scheduled for Monday or Tuesday, the issues are resolved by then!

**Reply by jcharnet** [reply]
Thanks for the update.
Tried v05.5-alpha.1 and now I'm seeing this error when I access 127.0.0.1:4455/dashboard

> {"error":{"code":500,"status":"Internal Server Error","message":"sqlite create: attempt to write a readonly database"}}

> kratos_1                      | time=2021-05-03T13:54:03Z level=info msg=completed handling request func=github.com/ory/x/reqlog.(*Middleware).ServeHTTP file=/go/pkg/mod/github.com/ory/x@v0.0.165/reqlog/middleware.go:139 method=GET name=public#http://127.0.0.1:4433/ remote=172.19.0.1:56394 request=/self-service/login/browser status=500 text_status=Internal Server Error took=50.873µs
> kratos-selfservice-ui-node_1  | GET /index.css 304 - - 3.395 ms
> kratos-selfservice-ui-node_1  | GET /form.css 304 - - 4.784 ms
> kratos-selfservice-ui-node_1  | GET /auth.css 304 - - 5.547 ms
> kratos-selfservice-ui-node_1  | GET /typography.css 304 - - 5.867 ms
>

**Reply by vinckr** [reply]
What exactly are the individual steps you did to get there? 
Is Docker running and docker compose installed? 
Your setup, additional log files and any other information that you think is helpful also helps us to solve your issue more quickly!

For me this works:
```
git clone https://github.com/ory/kratos.git
cd kratos
git checkout v0.5.5-alpha.1
docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate
```
Then open 127.0.0.1:4455/dashboard, ect.

**Reply by jcharnet** [reply]
It worked now, I cleaned up every docker image and volume, and now it worked. I believe it was using the same volume from my previous attempt with v0.6 and that could have been causing problems.

Thanks for all the responses.

**Comment by aeneasr** [comment, marked as answer]
I have fixed this on master!

**Reply by jcharnet** [reply]
Any cleanup I need to do? Got his error when running: 

>  docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate

> Pulling kratos-migrate (oryd/kratos:v0.6.0-alpha.1.pre.3-sqlite)...
> ERROR: manifest for oryd/kratos:v0.6.0-alpha.1.pre.3-sqlite not found: manifest unknown: manifest unknown
